envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/HttpIdleTimeoutTest.kt

```diff
@@ -27,57 +27,62 @@ internal class HttpIdleTimeoutTest : EnvoyControlTestConfiguration() {
     @Test
     fun `should close idle connections after 1s for HTTP2`() {
         // given
+        val name = "proxy1"
         registerService(name = "proxy1", port = EnvoyContainer.INGRESS_LISTENER_CONTAINER_PORT, container = envoyContainer2, tags = listOf("envoy"))
 
         // when
         untilAsserted {
-            callProxy()
+            callProxy(name)
         }
 
         // then
         untilAsserted {
-            assertHasActiveConnection(envoyContainer1)
+            assertHasActiveConnection(envoyContainer1, name)
         }
 
         // 5 seconds drain time + 10 idle + 5 padding
         untilAsserted(wait = Duration(20, TimeUnit.SECONDS)) {
-            assertHasNoActiveConnections(envoyContainer1)
+            assertHasNoActiveConnections(envoyContainer1, name)
         }
     }
 
     @Test
     fun `should close idle connections after 1s for HTTP1`() {
         // given
-        registerService(name = "proxy1", port = EnvoyContainer.INGRESS_LISTENER_CONTAINER_PORT, container = envoyContainer2)
+        val name = "proxy2"
+        registerService(name = name, port = EnvoyContainer.INGRESS_LISTENER_CONTAINER_PORT, container = envoyContainer2)
 
         // when
         untilAsserted {
-            callProxy()
+            callProxy(name)
         }
 
         // then
         untilAsserted {
-            assertHasActiveConnection(envoyContainer1)
+            assertHasActiveConnection(envoyContainer1, name)
         }
 
         // 10 idle + 5 padding
         untilAsserted(wait = Duration(15, TimeUnit.SECONDS)) {
-            assertHasNoActiveConnections(envoyContainer1)
+            assertHasNoActiveConnections(envoyContainer1, name)
         }
     }
 
-    private fun callProxy() {
-        val response = callService("proxy1")
+    private fun callProxy(name: String) {
+        val response = callService(name)
         assertThat(response).isOk()
     }
 
-    private fun assertHasActiveConnection(container: EnvoyContainer) {
-        val http2Connections = container.admin().statValue("cluster.proxy1.upstream_cx_active")?.toInt()
-        assertThat(http2Connections).isGreaterThan(0)
+    private fun assertHasActiveConnection(
+        container: EnvoyContainer,
+        name: String
+    ) {
+        val activeConnections = container.admin().statValue("cluster.$name.upstream_cx_active")?.toInt()
+        assertThat(activeConnections).isEqualTo(1)
```

Comment:
Why not `isGreaterThan(0)` ?

Reply:
It should be 1, I do not remember why originally I wrote `>0`.